spectrum/xpub.py

```diff
@@ -91,3 +127,22 @@ def __init__(self, driver):
 
     def follow(self):
         self._element.click()
+
+
+class XpubPeoplePicker():
+    CSS_PEOPLE_PICKER = '[data-test-id="people-picker-body"]'
+    CSS_PERSON_POD_BUTTON = '[data-test-id="person-pod-button"]'
+    CSS_ADD_BUTTON = '[data-test-id="people-picker-add"]'
+    TIMEOUT_CLOSING = 10
+
+    def __init__(self, driver):
+        self._driver = driver
+        self._picker = driver.find_element_by_css_selector(self.CSS_PEOPLE_PICKER)
+        self._add = driver.find_element_by_css_selector(self.CSS_ADD_BUTTON)
+
+    def choose_some(self, quantity):
+        buttons = self._picker.find_elements_by_css_selector(self.CSS_PERSON_POD_BUTTON)
+        for button in range(0, quantity):
+            buttons[button].click()
+        self._add.click()
+        WebDriverWait(self._driver, self.TIMEOUT_CLOSING).until_not(lambda driver: driver.find_element_by_css_selector(self.CSS_PEOPLE_PICKER).is_displayed())
```

Comment:
minor: Long line could do with prettifying

Reply:
Trying to shorten it altogether